packages/hash/frontend/src/pages/[account-slug]/entities/[entity-id].page/property-table.tsx

```diff
@@ -61,6 +62,7 @@ interface PropertyTableProps {
 
 type EnrichedPropertyType = PropertyType & {
   value: any;
+  /** @todo - Correct this, it is a property type BaseUri not an ID (it's unversioned) */
```

Comment:
@yusufkinatas I didn't want to change too much in the PR but we should try and distinguish between `baseUri`s and the `ID`s of types (`VersionedUri`s)

Reply:
And to flag that the most recent commit moves these changes and TODO https://github.com/hashintel/hash/pull/1220/commits/9ffa4cdf2315604e35cec858ea59e7df887b14be